runtime/common/src/constants.rs

```diff
@@ -283,12 +283,17 @@ impl Clone for MessagesMaxPayloadSizeBytes {
 }
 // -end- Messages Pallet ---
 
+// --- MSA Pallet ---
 /// The number of blocks per virtual bucket
 pub type MSAMortalityWindowSize = ConstU32<100>;
 /// The maximum number of signatures per virtual bucket
 pub type MSAMaxSignaturesPerBucket = ConstU32<50_000>;
 /// The total number of virtual buckets
 pub type MSANumberOfBuckets = ConstU32<2>;
+/// The upper limit on total stored signatures.
+/// This MUST be MaxSignaturesPerBucket * NumberOfBuckets.
+pub type MSAMaxSignaturesStored = ConstU32<100_000>;
```

Comment:
nit: do we really need this, since it is completely derivable from other configs?